buildchain/buildchain/image.py

```diff
@@ -79,10 +79,10 @@ def show() -> str:
             cmd='HARDLINK', width=constants.CMD_WIDTH,
             path=utils.build_relpath(constants.ISO_IMAGE_ROOT)
         )
-
-    task = targets.Target(task_dep=['_image_pull', '_image_build']).basic_task
+    task = targets.Target(task_dep=['check_for:hardlink',
```

Comment:
This can be wrapped as:

```
    task = targets.Target(
        task_dep=['_image_pull', '_image_build', 'check_for:hardlink']
    ).basic_task
```

which is a bit more readable.